day2OfJS/day2.js

```diff
@@ -0,0 +1,234 @@
+//Exercise: Level 1
+//1. Declare a variable named challenge and assign it to an initial value '30 Days Of JavaScript';
+const challenge = '30 Days Of JavaScript';
+console.log(challenge); //30 Days Of JavaScript
+
+
+//2. Print the string on the browser console using console.log()
+console.log('Hi there, my name is Luisa'); //Hi there, my name is Luisa
+
+
+//3. Print the length of the string on the browser console using console.log()
+const name = 'JavaScript';
+console.log(name.length); //9
+
+
+//4. Change all the string characters to capital letters using toUpperCase() method
+const mayus = 'JavaScript';
```

Comment:
A better fit for this variable name could be `stringToUpperCase`, remember to name the variables with a meaning of what are you doing